# 🛒 Superstore Sales Analysis

## 📌 Project Overview
This project analyzes the Superstore dataset to understand sales performance, profitability, customer behavior, and regional trends.

## 🎯 Objectives
- Analyze sales and profit performance
- Identify loss-making products and categories
- Understand the impact of discounts on profit
- Explore regional and customer trends

## 📁 Dataset Description

The dataset contains retail transaction data from a Superstore, including detailed information about orders, customers, products, and sales performance.

### Key Features:
- **Order Details:** Order ID, Order Date, Ship Date, Ship Mode  
- **Customer Information:** Customer ID, Customer Name, Segment  
- **Geographical Data:** Country, City, State, Region, Postal Code  
- **Product Information:** Product ID, Category, Sub-Category, Product Name  
- **Sales Metrics:** Sales, Quantity, Discount, Profit  

This dataset is used to analyze business performance, identify profit-driving areas, and uncover loss-making patterns.

In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

In [2]:
data = pd.read_csv('superstore.csv',encoding='latin1')
data.head(10)

,Row ID,Order ID,Order Date,Ship Date,Ship Mode,Customer ID,Customer Name,Segment,Country,City,...,Postal Code,Region,Product ID,Category,Sub-Category,Product Name,Sales,Quantity,Discount,Profit
0,1,CA-2016-152156,11/8/2016,11/11/2016,Second Class,CG-12520,Claire Gute,Consumer,United States,Henderson,...,42420,South,FUR-BO-10001798,Furniture,Bookcases,Bush Somerset Collection Bookcase,261.9600,2,0.00,41.9136
1,2,CA-2016-152156,11/8/2016,11/11/2016,Second Class,CG-12520,Claire Gute,Consumer,United States,Henderson,...,42420,South,FUR-CH-10000454,Furniture,Chairs,"Hon Deluxe Fabric Upholstered Stacking Chairs,...",731.9400,3,0.00,219.5820
2,3,CA-2016-138688,6/12/2016,6/16/2016,Second Class,DV-13045,Darrin Van Huff,Corporate,United States,Los Angeles,...,90036,West,OFF-LA-10000240,Office Supplies,Labels,Self-Adhesive Address Labels for Typewriters b...,14.6200,2,0.00,6.8714
3,4,US-2015-108966,10/11/2015,10/18/2015,Standard Class,SO-20335,Sean O'Donnell,Consumer,United States,Fort Lauderdale,...,33311,South,FUR-TA-10000577,Furniture,Tables,Bretford CR4500 Series Slim Rectangular Table,957.5775,5,0.45,-383.0310
4,5,US-2015-108966,10/11/2015,10/18/2015,Standard Class,SO-20335,Sean O'Donnell,Consumer,United States,Fort Lauderdale,...,33311,South,OFF-ST-10000760,Office Supplies,Storage,Eldon Fold 'N Roll Cart System,22.3680,2,0.20,2.5164
5,6,CA-2014-115812,6/9/2014,6/14/2014,Standard Class,BH-11710,Brosina Hoffman,Consumer,United States,Los Angeles,...,90032,West,FUR-FU-10001487,Furniture,Furnishings,Eldon Expressions Wood and Plastic Desk Access...,48.8600,7,0.00,14.1694
6,7,CA-2014-115812,6/9/2014,6/14/2014,Standard Class,BH-11710,Brosina Hoffman,Consumer,United States,Los Angeles,...,90032,West,OFF-AR-10002833,Office Supplies,Art,Newell 322,7.2800,4,0.00,1.9656
7,8,CA-2014-115812,6/9/2014,6/14/2014,Standard Class,BH-11710,Brosina Hoffman,Consumer,United States,Los Angeles,...,90032,West,TEC-PH-10002275,Technology,Phones,Mitel 5320 IP Phone VoIP phone,907.1520,6,0.20,90.7152
8,9,CA-2014-115812,6/9/2014,6/14/2014,Standard Class,BH-11710,Brosina Hoffman,Consumer,United States,Los Angeles,...,90032,West,OFF-BI-10003910,Office Supplies,Binders,DXL Angle-View Binders with Locking Rings by S...,18.5040,3,0.20,5.7825
9,10,CA-2014-115812,6/9/2014,6/14/2014,Standard Class,BH-11710,Brosina Hoffman,Consumer,United States,Los Angeles,...,90032,West,OFF-AP-10002892,Office Supplies,Appliances,Belkin F5C206VTEL 6 Outlet Surge,114.9000,5,0.00,34.4700


In [3]:
data.sample()

,Row ID,Order ID,Order Date,Ship Date,Ship Mode,Customer ID,Customer Name,Segment,Country,City,...,Postal Code,Region,Product ID,Category,Sub-Category,Product Name,Sales,Quantity,Discount,Profit
3554,3555,CA-2014-120838,3/23/2014,3/26/2014,Second Class,PO-18865,Patrick O'Donnell,Consumer,United States,Los Angeles,...,90036,West,OFF-ST-10000585,Office Supplies,Storage,Economy Rollaway Files,330.4,2,0.0,85.904


In [4]:
data.tail(5)

,Row ID,Order ID,Order Date,Ship Date,Ship Mode,Customer ID,Customer Name,Segment,Country,City,...,Postal Code,Region,Product ID,Category,Sub-Category,Product Name,Sales,Quantity,Discount,Profit
9989,9990,CA-2014-110422,1/21/2014,1/23/2014,Second Class,TB-21400,Tom Boeckenhauer,Consumer,United States,Miami,...,33180,South,FUR-FU-10001889,Furniture,Furnishings,Ultra Door Pull Handle,25.248,3,0.2,4.1028
9990,9991,CA-2017-121258,2/26/2017,3/3/2017,Standard Class,DB-13060,Dave Brooks,Consumer,United States,Costa Mesa,...,92627,West,FUR-FU-10000747,Furniture,Furnishings,Tenex B1-RE Series Chair Mats for Low Pile Car...,91.960,2,0.0,15.6332
9991,9992,CA-2017-121258,2/26/2017,3/3/2017,Standard Class,DB-13060,Dave Brooks,Consumer,United States,Costa Mesa,...,92627,West,TEC-PH-10003645,Technology,Phones,Aastra 57i VoIP phone,258.576,2,0.2,19.3932
9992,9993,CA-2017-121258,2/26/2017,3/3/2017,Standard Class,DB-13060,Dave Brooks,Consumer,United States,Costa Mesa,...,92627,West,OFF-PA-10004041,Office Supplies,Paper,"It's Hot Message Books with Stickers, 2 3/4"" x 5""",29.600,4,0.0,13.3200
9993,9994,CA-2017-119914,5/4/2017,5/9/2017,Second Class,CC-12220,Chris Cortes,Consumer,United States,Westminster,...,92683,West,OFF-AP-10002684,Office Supplies,Appliances,"Acco 7-Outlet Masterpiece Power Center, Wihtou...",243.160,2,0.0,72.9480


1. Data Understanding (Start Strong)

    1. What is the size of the dataset? (number of rows and columns)

In [5]:
data.shape

(9994, 21)

number of rows = 9994

number of columns = 21


2. What are the features (columns) and their data types in the dataset?

In [6]:
data.columns

Index(['Row ID', 'Order ID', 'Order Date', 'Ship Date', 'Ship Mode',
       'Customer ID', 'Customer Name', 'Segment', 'Country', 'City', 'State',
       'Postal Code', 'Region', 'Product ID', 'Category', 'Sub-Category',
       'Product Name', 'Sales', 'Quantity', 'Discount', 'Profit'],
      dtype='object')

In [7]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 9994 entries, 0 to 9993
Data columns (total 21 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Row ID         9994 non-null   int64  
 1   Order ID       9994 non-null   object 
 2   Order Date     9994 non-null   object 
 3   Ship Date      9994 non-null   object 
 4   Ship Mode      9994 non-null   object 
 5   Customer ID    9994 non-null   object 
 6   Customer Name  9994 non-null   object 
 7   Segment        9994 non-null   object 
 8   Country        9994 non-null   object 
 9   City           9994 non-null   object 
 10  State          9994 non-null   object 
 11  Postal Code    9994 non-null   int64  
 12  Region         9994 non-null   object 
 13  Product ID     9994 non-null   object 
 14  Category       9994 non-null   object 
 15  Sub-Category   9994 non-null   object 
 16  Product Name   9994 non-null   object 
 17  Sales          9994 non-null   float64
 18  Quantity

3. What does each column represent? (e.g., Order ID, Order Date, Ship Mode, Sales, Profit, etc.)
* -> Each column represent the flowing information:-
* 0   Row ID         - Unique ID for each row.
* 1   Order ID       - Unique Order ID for each Customer.
* 2   Order Date     - Order Date of the product.
* 3   Ship Date      - Shipping Date of the Product.
* 4   Ship Mode      - Shipping Mode specified by the Customer.
* 5   Customer ID    - Unique ID to identify each Customer.
* 6   Customer Name  - Name of the Customer.
* 7   Segment        - The segment where the Customer belongs.
* 8   Country        - Country of residence of the Customer.
* 9   City           - City of residence of of the Customer.
* 10  State          - State of residence of the Customer.
* 11  Postal Code    - Postal Code of every Customer.
* 12  Region         - Region where the Customer belong.
* 13  Product ID     - Unique ID of the Product.
* 14  Category       - Category of the product ordered.
* 15  Sub-Category   - Sub-Category of the product ordered.
* 16  Product Name   - Name of the Product
* 17  Sales          - Sales of the Product.
* 18  Quantity       - Quantity of the Product.
* 19  Discount       - Discount provided.
* 20  Profit         - Profit/Loss incurred.

4. Which features are categorical (e.g., Category, Region, Segment) and which are numerical (e.g., Sales, Profit, Quantity)?

In [8]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 9994 entries, 0 to 9993
Data columns (total 21 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Row ID         9994 non-null   int64  
 1   Order ID       9994 non-null   object 
 2   Order Date     9994 non-null   object 
 3   Ship Date      9994 non-null   object 
 4   Ship Mode      9994 non-null   object 
 5   Customer ID    9994 non-null   object 
 6   Customer Name  9994 non-null   object 
 7   Segment        9994 non-null   object 
 8   Country        9994 non-null   object 
 9   City           9994 non-null   object 
 10  State          9994 non-null   object 
 11  Postal Code    9994 non-null   int64  
 12  Region         9994 non-null   object 
 13  Product ID     9994 non-null   object 
 14  Category       9994 non-null   object 
 15  Sub-Category   9994 non-null   object 
 16  Product Name   9994 non-null   object 
 17  Sales          9994 non-null   float64
 18  Quantity

numerical column:-
* Row ID        int64
* Postal Code   int64  
* Sales         float64
* Quantity      int64  
* Discount      float64
* Profit        float64

and all other columns are categorical


5. What is the time range of the dataset based on Order Date?

In [9]:
# 1. Convert 'Order Date' to datetime format
data['Order Date'] = pd.to_datetime(data['Order Date'])

# 2. Find the minimum (earliest) and maximum (latest) dates
start_date = data['Order Date'].min()
end_date = data['Order Date'].max()

# 3. Calculate the total range
date_range = (end_date - start_date).days

print(f"Dataset Start Date: {start_date.strftime('%Y-%m-%d')}")
print(f"Dataset End Date:   {end_date.strftime('%Y-%m-%d')}")
print(f"Total date Range:   {date_range} days")

Dataset Start Date: 2014-01-03
Dataset End Date:   2017-12-30
Total date Range:   1457 days


🧹 2. Data Cleaning & Preprocessing

6. Are there any missing values in columns like Postal Code, Sales, Profit, etc.? How did you handle them?

In [10]:
data.isna().sum()

Row ID           0
Order ID         0
Order Date       0
Ship Date        0
Ship Mode        0
Customer ID      0
Customer Name    0
Segment          0
Country          0
City             0
State            0
Postal Code      0
Region           0
Product ID       0
Category         0
Sub-Category     0
Product Name     0
Sales            0
Quantity         0
Discount         0
Profit           0
dtype: int64

No any missing values. so no need to handle.

7. Are there any duplicate rows based on Order ID or full records?

In [11]:
data.duplicated().sum()

np.int64(0)

No any duplicated data.

8. Are data types correct? (e.g., convert Order Date and Ship Date to datetime)

In [12]:
data['Ship Date']

0       11/11/2016
1       11/11/2016
2        6/16/2016
3       10/18/2015
4       10/18/2015
           ...    
9989     1/23/2014
9990      3/3/2017
9991      3/3/2017
9992      3/3/2017
9993      5/9/2017
Name: Ship Date, Length: 9994, dtype: object

No data type for ship date is not correct.

In [13]:
# Convert 'Order Date' to datetime format
data['Order Date'] = pd.to_datetime(data['Order Date'])
data['Ship Date'] = pd.to_datetime(data['Ship Date'])

In [14]:
data['Order Date']

0      2016-11-08
1      2016-11-08
2      2016-06-12
3      2015-10-11
4      2015-10-11
          ...    
9989   2014-01-21
9990   2017-02-26
9991   2017-02-26
9992   2017-02-26
9993   2017-05-04
Name: Order Date, Length: 9994, dtype: datetime64[ns]

9. Are there outliers in Sales, Profit, or Quantity?

In [15]:
# function to check and handle outliers
def handle_outliers(df, column_name):
    # 1. Calculate IQR statistics using clean, local variable names
    Q1 = df[column_name].quantile(0.25)
    Q3 = df[column_name].quantile(0.75)
    IQR = Q3 - Q1
    
    # 2. Define bounds
    lower_bound = Q1 - 1.5 * IQR
    upper_bound = Q3 + 1.5 * IQR
    
    # 3. Find and count outliers
    outliers_mask = (df[column_name] < lower_bound) | (df[column_name] > upper_bound)
    outlier_count = outliers_mask.sum()
    print(f"Found {outlier_count} outliers in '{column_name}'")
    
    # # 4. Replace outliers with the column median
    # median_val = df[column_name].median()
    # df.loc[outliers_mask, column_name] = median_val
    
    return df


In [16]:
data = handle_outliers(data, 'Sales')
data = handle_outliers(data, 'Profit')
data = handle_outliers(data, 'Quantity')

Found 1167 outliers in 'Sales'
Found 1881 outliers in 'Profit'
Found 170 outliers in 'Quantity'


10. Did you create new features? (e.g., Profit Ratio, Order Year, Order Month, Shipping Duration)

In [17]:
# 1. Shipping Duration (Days between order and shipping)
data['Shipping Duration'] = (data['Ship Date'] - data['Order Date']).dt.days

# 2. Order Year
data['Order Year'] = data['Order Date'].dt.year

# 3. Order Month (Name format like 'January' or number format like 1)
data['Order Month'] = data['Order Date'].dt.strftime('%B')

# 4. Profit Ratio (Profit divided by Total Sales)
data['Profit Ratio'] = data['Profit'] / data['Sales']

In [18]:
data.sample(4)

,Row ID,Order ID,Order Date,Ship Date,Ship Mode,Customer ID,Customer Name,Segment,Country,City,...,Sub-Category,Product Name,Sales,Quantity,Discount,Profit,Shipping Duration,Order Year,Order Month,Profit Ratio
1865,1866,US-2017-116659,2017-11-12,2017-11-12,Same Day,NG-18355,Nat Gilpin,Corporate,United States,Newark,...,Phones,Jabra SPEAK 410 Multidevice Speakerphone,370.7820,3,0.40,-92.6955,0,2017,November,-0.250000
765,766,CA-2014-162775,2014-01-13,2014-01-15,Second Class,CS-12250,Chris Selesnick,Corporate,United States,Bossier City,...,Accessories,Plantronics S12 Corded Telephone Headset System,646.7400,6,0.00,258.6960,2,2014,January,0.400000
3023,3024,CA-2017-124674,2017-11-17,2017-11-23,Standard Class,JB-16000,Joy Bell-,Consumer,United States,Brownsville,...,Bookcases,"Atlantic Metals Mobile 2-Shelf Bookcases, Cust...",327.7328,2,0.32,-14.4588,6,2017,November,-0.044118
3414,3415,CA-2016-100244,2016-09-20,2016-09-24,Standard Class,GM-14695,Greg Maxwell,Corporate,United States,San Francisco,...,Binders,Ibico Plastic Spiral Binding Combs,48.6400,2,0.20,15.8080,4,2016,September,0.325000


📊 3. Exploratory Data Analysis (EDA)

11. What are the total Sales, total Profit, and total number of Orders?

In [19]:
print("Total sales = ",data['Sales'].sum())
print(f"total profit = {data['Profit'].sum()}")
print(f"Total number of order = {data['Order ID'].count()}")

Total sales =  2297200.8603
total profit = 286397.0217
Total number of order = 9994


12. What is the average sales per order (Average Order Value)?

In [20]:
# 1. Group by 'Order ID' to sum up the sales for each unique order
sales_per_order = data.groupby('Order ID')['Sales'].sum()

# 2. Calculate the average of those order totals
average_order_value = sales_per_order.mean()

print(f"Average Order Value: ${average_order_value:,.2f}")

Average Order Value: $458.61


13. How are Sales and Profit distributed across the dataset?

In [21]:
data[['Sales','Profit']].describe()

,Sales,Profit
count,9994.000000,9994.000000
mean,229.858001,28.656896
std,623.245101,234.260108
min,0.444000,-6599.978000
25%,17.280000,1.728750
50%,54.490000,8.666500
75%,209.940000,29.364000
max,22638.480000,8399.976000


14. Which products (Product Name) are most frequently sold (based on Quantity)?

In [22]:
data.sample()

,Row ID,Order ID,Order Date,Ship Date,Ship Mode,Customer ID,Customer Name,Segment,Country,City,...,Sub-Category,Product Name,Sales,Quantity,Discount,Profit,Shipping Duration,Order Year,Order Month,Profit Ratio
5456,5457,CA-2017-130631,2017-12-29,2018-01-02,Standard Class,BS-11755,Bruce Stewart,Consumer,United States,Edmonds,...,Fasteners,Acco Glide Clips,19.6,5,0.0,9.604,4,2017,December,0.49


In [23]:
data['Product Name'].value_counts().head(5)

Product Name
Staple envelope            48
Staples                    46
Easy-staple paper          46
Avery Non-Stick Binders    20
Staples in misc. colors    19
Name: count, dtype: int64

these are top 5 most sold product names.

In [24]:
data['Product Name'].value_counts().tail(5)

Product Name
Boston 1900 Electric Pencil Sharpener                                                  1
RCA ViSYS 25423RE1 Corded phone                                                        1
Canon Color ImageCLASS MF8580Cdw Wireless Laser All-In-One Printer, Copier, Scanner    1
Newell 342                                                                             1
Eldon Jumbo ProFile Portable File Boxes Graphite/Black                                 1
Name: count, dtype: int64

these are top 5 least sold product name.

💰 4. Sales Analysis

15. Which Category (Furniture, Office Supplies, Technology) generates the highest sales?

In [25]:
data.sample()

,Row ID,Order ID,Order Date,Ship Date,Ship Mode,Customer ID,Customer Name,Segment,Country,City,...,Sub-Category,Product Name,Sales,Quantity,Discount,Profit,Shipping Duration,Order Year,Order Month,Profit Ratio
8549,8550,CA-2015-126186,2015-09-11,2015-09-12,First Class,GB-14530,George Bell,Corporate,United States,Aurora,...,Furnishings,Deflect-O Glasstique Clear Desk Accessories,24.64,4,0.2,4.004,1,2015,September,0.1625


In [26]:
data.groupby('Category')['Sales'].mean().sort_values(ascending=False)

Category
Technology         452.709276
Furniture          349.834887
Office Supplies    119.324101
Name: Sales, dtype: float64

Technology category generates the highest sales.

16. Which Sub-Category contributes the most to sales?

In [27]:
data.groupby('Sub-Category')['Sales'].mean().sort_values(ascending=False).head(10)

Sub-Category
Copiers        2198.941618
Machines       1645.553313
Tables          648.794771
Chairs          532.332420
Bookcases       503.859633
Phones          371.211534
Storage         264.590553
Supplies        245.650200
Appliances      230.755710
Accessories     215.974604
Name: Sales, dtype: float64

these are the top 10 Sub-Category that contribute the most to sales.

17. What are the top 10 products by total Sales?

In [28]:
data.groupby('Product Name')['Sales'].sum().sort_values(ascending=False).head(10)

Product Name
Canon imageCLASS 2200 Advanced Copier                                          61599.824
Fellowes PB500 Electric Punch Plastic Comb Binding Machine with Manual Bind    27453.384
Cisco TelePresence System EX90 Videoconferencing Unit                          22638.480
HON 5400 Series Task Chairs for Big and Tall                                   21870.576
GBC DocuBind TL300 Electric Binding System                                     19823.479
GBC Ibimaster 500 Manual ProClick Binding System                               19024.500
Hewlett Packard LaserJet 3310 Copier                                           18839.686
HP Designjet T520 Inkjet Large Format Printer - 24" Color                      18374.895
GBC DocuBind P400 Electric Binding System                                      17965.068
High Speed Automatic Electric Letter Opener                                    17030.312
Name: Sales, dtype: float64

Top 10 products wtih highes sales are given above.

18. Which products have the lowest sales?

In [29]:
data.groupby('Product Name')['Sales'].sum().sort_values(ascending=True).head(10)

Product Name
Eureka Disposable Bags for Sanitaire Vibra Groomer I Upright Vac    1.624
Avery 5                                                             5.760
Xerox 20                                                            6.480
Grip Seal Envelopes                                                 7.072
Avery Hi-Liter Pen Style Six-Color Fluorescent Set                  7.700
Avery Hi-Liter Comfort Grip Fluorescent Highlighter, Yellow Ink     7.800
Xerox 1989                                                          7.968
4009 Highlighters                                                   8.040
Stockwell Gold Paper Clips                                          8.096
Newell 308                                                          8.400
Name: Sales, dtype: float64

these are the top 10 product with lowest sales.

📉 5. Profit Analysis (VERY IMPORTANT)

19. Which Category is the most profitable?

In [31]:
data.groupby('Category')['Profit'].sum().sort_values(ascending=False)

Category
Technology         145454.9481
Office Supplies    122490.8008
Furniture           18451.2728
Name: Profit, dtype: float64

Technology Category is the most profitable.

20. Which Sub-Category is loss-making (negative profit)?

In [34]:
data.groupby('Sub-Category')['Profit'].sum().sort_values(ascending=True).head()

Sub-Category
Tables      -17725.4811
Bookcases    -3472.5560
Supplies     -1189.0995
Fasteners      949.5182
Machines      3384.7569
Name: Profit, dtype: float64

sub-category Tables, Bookcases and Supplies is loss making.

21. What are the top 10 most profitable products?

In [38]:
data.groupby('Product Name')['Profit'].sum().sort_values(ascending=False).head(10)

Product Name
Canon imageCLASS 2200 Advanced Copier                                          25199.9280
Fellowes PB500 Electric Punch Plastic Comb Binding Machine with Manual Bind     7753.0390
Hewlett Packard LaserJet 3310 Copier                                            6983.8836
Canon PC1060 Personal Laser Copier                                              4570.9347
HP Designjet T520 Inkjet Large Format Printer - 24" Color                       4094.9766
Ativa V4110MDD Micro-Cut Shredder                                               3772.9461
3D Systems Cube Printer, 2nd Generation, Magenta                                3717.9714
Plantronics Savi W720 Multi-Device Wireless Headset System                      3696.2820
Ibico EPK-21 Electric Binding System                                            3345.2823
Zebra ZM400 Thermal Label Printer                                               3343.5360
Name: Profit, dtype: float64

Top 10 poduct name with most profit ara given above.

22. Which products have high sales but low or negative profit?

In [46]:
# 1. Group and sum the columns
grouped = data.groupby('Product Name')[['Sales', 'Profit']].sum()

# 2. Filter for negative profit, then sort by Sales descending
grouped[grouped['Profit'] < 0].sort_values(by='Sales', ascending=False).head(10)

,Sales,Profit
Product Name,,
Cisco TelePresence System EX90 Videoconferencing Unit,22638.4800,-1811.0784
GBC DocuBind P400 Electric Binding System,17965.0680,-1878.1662
High Speed Automatic Electric Letter Opener,17030.3120,-262.0048
Lexmark MX611dhe Monochrome Laser Printer,16829.9010,-4589.9730
Martin Yale Chadless Opener Electric Letter Opener,16656.2000,-1299.1836
"Riverside Palais Royal Lawyers Bookcase, Royale Cherry Finish",15610.9656,-669.5448
Bretford Rectangular Conference Table Tops,12995.2915,-327.2331
Cubify CubeX 3D Printer Double Head Print,11099.9630,-8879.9704
Chromcraft Bull-Nose Wood Oval Conference Tables & Bases,9917.6400,-2876.1156


these are the top 10 product names that has the highest sales but also loss-making.